lib/rollbax/item.ex

```diff
@@ -15,20 +15,75 @@ defmodule Rollbax.Item do
     }
   end
 
-  def compose(draft, {level, msg, time, meta}) do
+  def compose(draft, {level, exception, stacktrace, time, meta}) do
     {occurr_data, meta} =
-      Map.pop(meta, :rollbax_occurr_data, %{})
+      Map.pop(meta, "rollbax_occurr_data", %{})
```

Comment:
This isn't going to be part of the resulting JSON. In fact, we use the `:rollbax_occurr_data` atom to store this data as you can see [here](https://github.com/elixir-addicts/rollbax/blob/455c86812cd49607dbd9cad501de765163099fdd/lib/rollbax.ex#L34). So, let's use an atom here instead of a string :)